packages/infrastructure/authentik_github_sso/README.md

```diff
@@ -0,0 +1,53 @@
+import MarkdownAlert from "@/components/markdown/MarkdownAlert.astro";
+
+# Authentik GitHub SSO
+
+This module configures Authentik for integration with GitHub SAML single sign-on.
+
+***Note:*** The [GitHub Enterprise plan](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-iam/using-saml-for-enterprise-iam/configuring-saml-single-sign-on-for-your-enterprise) is required for SSO.   
+
+<MarkdownAlert severity="warning">
+  Due to limitations with GitHub, this module only handles authentication and does not support user provisioning or de-provisioning at this time. As a result, users will not be automatically created or removed from GitHub when they are added or removed from Authentik.
+
+  When a user is removed from Authentik, they will lose access to the organization. However, be aware of the following caveats:
+  - Any active session tokens that the user has with the GitHub web UI and PATs the user may have generated will not be automatically revoked. Until these tokens expire, the user may still interact with the web UI / API unless they are manually removed from the GitHub organization.
+</MarkdownAlert>
+
+## Guide
+
+### Start the GitHub SAML SSO Setup
+
+1. Log in to GitHub and navigate to your enterprise's dashboard. For example, Panfactum's enterprise url is https://github.com/enterprises/Panfactum.
```

Comment:
Good to have an example, but personally it was extremely difficult to find out how to do this. I can imagine this being a blocker for less motivated folks. Can we add clarity on the recommended path to find this.